### DISCUSSION TITLE
What happened to https://github.com/wercker/stern ?

### DISCUSSION THREAD

**Discussion by vilaca** [stern/stern#198] [General]
I maintain https://github.com/vilaca/awesome-k8s-tools and just found out https://github.com/wercker/stern and https://github.com/wercker are gone.

**Comment by superbrothers** [comment]
Since stern development stopped after Wercker was acquired by Oracle, we (a group of individuals) forked and maintain it.

**Reply by vilaca** [reply]
Thank you for keeping the project alive. I use it daily.

**Reply by floryut** [reply]
It's a pretty new thing to have the Wercker project and group entirely removed though, surprise to see this.
